src/modules/transaction/components/TransactionPriority/TransactionPriority.css

```diff
@@ -83,6 +83,29 @@
 
       & .fieldLabel {
         display: inline-block;
+
+        & div > div {
+          width: 18vw !important; /* stylelint-disable-line */
```

Comment:
<img width="749" alt="image" src="https://user-images.githubusercontent.com/8784876/227157026-6efc26e4-fbf0-414f-9d90-a9d20c2c85cb.png">

probably need 16vw.
Also, no need for !important

Reply:
@oskarleonard Fixed